pasta_eln/dataverse/dataset-create-new-all-default-fields.json

```diff
@@ -1417,7 +1417,7 @@
             ]
           },
           {
-            "typeName": "studyAssayOtherMeasurmentType",
```

Comment:
Please do not change this schema file, since it's provided by data-verse and can lead to REST API failure!